packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3675,3 +3677,15 @@ class _SemanticsGeometry {
   bool get markAsHidden => _markAsHidden;
   bool _markAsHidden = false;
 }
+
+/// A class that creates [DiagnosticsNode] by wrapping [RenderObject.debugCreator]
+class DiagnosticsDebugCreator extends DiagnosticsProperty<Object> {
+  /// Create a [DiagnosticsNode]
```

Comment:
this documentation violates our style guide (it doesn't say anything that isn't obvious)